eli5/sklearn/explain_weights.py

```diff
@@ -412,3 +414,19 @@ def _features(target_id):
             method='linear model',
             is_regression=True,
         )
+
+
+@explain_weights.register(Pipeline)
+def explain_weights_pipeline(estimator, feature_names=None, **kwargs):
+    last_estimator = estimator.steps[-1][1]
+    transform_pipeline = Pipeline(estimator.steps[:-1])
+    if hasattr(feature_names, 'get_feature_names'):
```

Comment:
This is to reflect some of the behaviour of `eli5.sklearn.utils.get_feature_names`. If you don't care for it, I'll remove it.